code/game/gamemodes/clock_cult/clock_items/ratvarian_spear.dm

```diff
@@ -38,11 +38,22 @@
 		clockwork_desc = "A powerful spear of Ratvarian making. It's more effective against enemy cultists and silicons, though it won't last for long."
 		timerid = addtimer(src, "break_spear", 600, TIMER_NORMAL)
 
+/obj/item/clockwork/ratvarian_spear/cyborg/ratvar_act() //doesn't break!
+	if(ratvar_awakens)
+		force = 25
+		throwforce = 50
```

Comment:
why set throwforce, borgs can't throw?

or am i out of the loop and borgs can throw spears because snowflake